tests/sles4sap/bone_install.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2019-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install SAP Business One via command line.
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'sles4sap';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use Utils::Backends;
+use utils qw(file_content_replace zypper_call);
+use Utils::Systemd 'systemctl';
+use version_utils 'is_sle';
+use POSIX 'ceil';
+use Utils::Logging 'save_and_upload_log';
+
+sub run {
+    my ($self) = @_;
+    my ($proto, $path) = $self->fix_path(get_required_var('BONE'));
+    my $sid = get_required_var('INSTANCE_SID');
+    my $instid = get_required_var('INSTANCE_ID');
+    # set timeout as 4800 as a temp workaround for slow nfs
+    my $tout = get_var('HANA_INSTALLATION_TIMEOUT', 4800);    # Timeout for HANA installation commands.
+
+    select_serial_terminal;
+
+    # Transfer media.
+    my $target = '/sapinst';    # Directory in SUT where install media will be copied token
+    if (get_var 'ASSET_0') {
+        # If the ASSET_0 variable is defined, the test will attempt to download
+        # the HANA media from the factory/other directory of the openQA server.
+        record_info "Dowloading using ASSET_0";
+        $self->download_hana_assets_from_server(target => $target, nettout => $tout);
+    }
+    elsif (get_required_var 'BONE') {
+        # If not, the media will be retrieved from a remote server.
+        record_info "Downloading using $proto";
+        $self->copy_media($proto, $path, $tout, $target);
+    }
+
+    # install bin is used for installation, verify if it exists
+    my $install_bin = '/sapinst/' . get_var('B1_INSTALL', "B1/Packages.Linux/ServerComponents" . "/install");
```

Comment:
```suggestion
    my $install_bin = join('/',$target,get_var('B1_INSTALL', 'B1/Packages.Linux/ServerComponents/install');
```

Reply:
Why there's this concatenation between two strings (no variable formatting is going on, just two constant strings)?
```
"B1/Packages.Linux/ServerComponents" . "/install"
```

Reply:
done